tpm2/tpm2.go

```diff
@@ -1327,10 +1327,10 @@ func decodeSign(buf []byte) (*Signature, error) {
 	return DecodeSignature(in)
 }
 
-// Sign computes a signature for digest using a given loaded key. Signature
+// SignWithSession computes a signature for digest using a given loaded key. Signature
```

Comment:
This comment shouldn't be exactly identical, at least explain the difference between it and `Sign`.

Reply:
Added more descriptive comment